modules/apps/portal-store/portal-store-s3-test/src/test/java/com/liferay/portal/store/s3/S3FileCacheCleanUpTest.java

```diff
@@ -159,6 +186,13 @@ public void testExpiredCacheDirWithSingleFileAndValidSubdirectoryWithSingleFile(
 		Path subdirCachedFilePath = Files.createTempFile(
 			subdirPath, "subdirCachedFile", null);
 
+		long currentTime = System.currentTimeMillis();
+
+		Files.setLastModifiedTime(
+			_cacheDirPath, FileTime.fromMillis(currentTime - 5000));
+		Files.setLastModifiedTime(
```

Comment:
@ericyanLr this can not be right, you are making this file timed out, and still asserting it exist later.